Components/Communication/Inc/UARTTask.hpp

```diff
@@ -22,38 +20,33 @@ enum UART_TASK_COMMANDS {
     UART_TASK_COMMAND_MAX
 };
 
-
 /* Class ------------------------------------------------------------------*/
-class UARTTask : public Task
-{
-public:
+class UARTTask : public Task {
+   public:
     static UARTTask& Inst() {
         static UARTTask inst;
         return inst;
     }
 
     void InitTask();
 
-protected:
-    static void RunTask(void* pvParams) { UARTTask::Inst().Run(pvParams); } // Static Task Interface, passes control to the instance Run();
+   protected:
+    static void RunTask(void* pvParams) {
+        UARTTask::Inst().Run(pvParams);
+    }  // Static Task Interface, passes control to the instance Run();
```

Comment:
If possible, try to make functions that are a single line inside the class definition stay a single line... though that's oddly specific so 🤷

Reply:
It will do this already if it does not exceed the column limit. The issue here is the comment exceeds the limit.